.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Modules/IntelliTect.PSDropbin"]
 	path = Modules/IntelliTect.PSDropbin
 	url = https://github.com/IntelliTect/PSDropbin.git
+[submodule "submodules/RamblingCookieMonster/PowerShell"]
```

Comment:
Is there a way we could take a dependency on https://www.powershellgallery.com/packages/WFTools/0.1.58 rather than bringing it in as a submodule?

Reply:
@adamskt ... Good point regarding New-DynamicParam. The alternative, would be to add the submodule from WFTools as you suggest? At the moment, I don't believe we are exporting it as a command. However, clearly, we are distributing it. Suggestions?

There is lots of other unrelated stuff in WFTools... I'm not sure we want to be redistributing it. Thoughts?